modules/express/src/typedRoutes/api/v2/ofcSignPayload.ts

```diff
@@ -0,0 +1,31 @@
+import * as t from 'io-ts';
+import { Json, NonEmptyString } from 'io-ts-types';
+import { httpRoute, httpRequest, optional } from '@api-ts/io-ts-http';
+
+/**
+ * Sign an arbitrary payload using an OFC trading account key.
+ * @operationId express.ofc.signPayload
+ * POST /api/v2/ofc/signPayload
+ */
+export const OfcSignPayloadBody = {
+  walletId: NonEmptyString,
+  payload: Json,
+  walletPassphrase: optional(t.string),
+};
+
+/**
+ * Request body for signing an arbitrary payload with an OFC wallet key.
+ * @property walletId – non-empty string identifying the OFC wallet
+ * @property payload  – JSON value to sign
+ * @property walletPassphrase – optional passphrase used to decrypt the user key (string)
+ */
+export const PostOfcSignPayload = httpRoute({
+  path: '/api/v2/ofc/signPayload',
+  method: 'POST',
+  request: httpRequest({ body: OfcSignPayloadBody }),
+  response: {
+    200: t.type({ payload: t.string, signature: t.string }),
```

Comment:
Please define the response codec as a variable and add some jsdoc's regarding their fields. 

The codecs need to be their own variables due to how the code generation works.

Reply:
Also to ensure this works as we expect it to, lets find an existing test for `signPayload` and use the codec to decode the response